Gemfile

```diff
@@ -78,3 +78,6 @@ end
 
```

Comment:
![image](https://github.com/microhass/blog-app/assets/93039359/ec3bc3bf-349d-4dc9-85c8-574864afcd0f)

- [ ] It would be nice if you removed the Getting Started section at the top of the readme file because you already have a Getting Started section after the Key Features section so it would be great if you remove the Getting Started which is at the top of the readme to make your readme descriptive and professional 👍